src/sentry/integrations/slack/event_endpoint.py

```diff
@@ -164,34 +205,60 @@ def on_message(self, request, integration, token, data):
 
         return self.respond()
 
+    def _parse_url(self, link) -> Union[Tuple[LinkType, Mapping[str, Any]], Tuple[None, None]]:
+        """
+        Determines if the URL is something we are able to unfurl, if it is we
+        will return the named groups from the regex match, along with the link
+        type.
+        """
+        link_type, match = match_link(link)
+
+        if link_type is None or match is None:
+            return None, None
```

Comment:
@EvanPurkhiser what happened to returning the specific event ID? I think we still want to be able to unfurl to a specific event.

Reply:
It does that, it's one of the regex groups that is part of the match